src/api/actions/checkContentModeration.ts

```diff
@@ -0,0 +1,43 @@
+import { Maybe } from "../../utilities";
+import { ContentModeration, ContentModerationResponse } from "../../types";
+import { makeApiCall } from "./helpers";
+
+const regex = /[^\w\s]/; //Matches whitespace and alphanumeric characters
+const reasons = {
+  specialCharacters: "Contains special characters.",
+  explicitContent: "Contains explicit content."
+}
+export const checkContentModeration = async (
+  apiUri: string,
+  text: string
+): Promise<ContentModerationResponse> => {
+  let moderation: Maybe<ContentModeration>;
+  let response: ContentModerationResponse = {
```

Comment:
I may be totally wrong but response could be const?

Reply:
Made object const